### ISSUE TITLE
Make it so that removing wheels of military vehicles is illegal

### ISSUE THREAD

**Issue by SirGhosty22** [Sparker95/Vindicta#206]
Just tried this trick and managed to get the ai to dismount a gorgon, it seems that very few missions do this, and this being added would be dope.

**Comment by Sparker95** [comment]
Good idea, thx. The bots should have repaired the wheel btw when they are not in combat any more.

**Comment by SEBArma** [comment]
make ace-interaction with any military asset in general illigal - that would als prevent people from being able to use ace medical overdoses to kill soldiers covertly

**Comment by Sparker95** [comment]
Would be cool if someone looked up names of all these event handlers so that I can add them.
Ace exposes some event handlers for such things.

**Comment by SEBArma** [comment]
i would wait for the medical rewrite update to drop before attemting this, there will be significant changes https://ace3mod.com/2019/12/18/medical-rewrite.html

**Comment by Sparker95** [comment]
Yes but non-medical event names should stay the same I think

**Comment by zalexki** [comment]
@marvis999 didnt you implement this already ?

**Comment by zalexki** [comment]
Linked to #261
